spacy/ml/tb_framework.pyx

```diff
@@ -183,23 +183,23 @@ def _forward_greedy_cpu(model: Model, TransitionSystem moves, states: List[State
     for state in states:
         if not state.is_final():
             c_states.push_back(state.c)
-    weights = get_c_weights(model, <float*>feats.data, seen_mask)
+    weights = _get_c_weights(model, <float*>feats.data, seen_mask)
     # Precomputed features have rows for each token, plus one for padding.
     cdef int n_tokens = feats.shape[0] - 1
-    sizes = get_c_sizes(model, c_states.size(), n_tokens)
+    sizes = _get_c_sizes(model, c_states.size(), n_tokens)
```

Comment:
What does the `c` in `_get_c_weights/sizes` signify? Is it just a shorthand for `class` (in which case, I think we can probably do away with it entirely)?

Reply:
This comes from the old parser, but I think `c_` here means: lives in C-land as opposed to Python-land.

Reply:
I do see an upside to the naming, since `tb_framework` contains both Thinc models (which we could consider being Python-land) and the fast C path.